README.md

```diff
@@ -16,11 +16,11 @@ To check out any local changes you've made:
 
 A local server should start and you should be able to access a local version of the website from there.
 
-* Note: If you try to access tab assets, and recover 404 Not Found, it's necessary to into the repository .\generate-assets and run the command in terminal: <br>
+## Assets Page
```

Comment:
```suggestion
### Assets Page
```
I think this should be a sub section of the Zola build instructions

Reply:
It's just a way to jump line, but I put in the format of the upper topic.

Reply:
Done 👍